sdk/keyvault/Azure.Security.Keyvault.Keys/src/Azure.Security.Keyvault.Keys.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>This is the Microsoft Azure Key Vault Keys client library</Description>
+    <AssemblyTitle>Microsoft Azure.Security.KeyVault.Keys client library</AssemblyTitle>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <PackageTags>Microsoft Azure Key Vault Keys</PackageTags>
+    <PackageReleaseNotes>
+      <![CDATA[
+      ]]>
+    </PackageReleaseNotes>
+
+    <!-- Make sure that we don't pull in additional dependencies during build or package -->
+    <ImportDefaultReferences>false</ImportDefaultReferences>
+
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <NoWarn>$(NoWarn);3021</NoWarn>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\core\Azure.Core\src\Azure.Core.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Memory" Version="4.5.2" />
+    <PackageReference Include="Microsoft.Bcl.Json.Sources" Version="4.6.0-preview3.19128.7">
```

Comment:
This is an old package, we've moved to using `<PackageReference Include="System.Text.Json" Version="4.6.0-preview6.19226.8" />`

Reply:
```
Warning	NU1603	[...] System.Text.Json 4.6.0-preview6.19226.8 was not found. An approximate best match of System.Text.Json 4.6.0-preview6.19259.10 was resolved.
```
I updated the reference to `System.Text.Json 4.6.0-preview6.19259.10`